.github/workflows/js-test-and-release.yml

```
@@ -0,0 +1,131 @@
+name: test & maybe release
+on:
+  push:
+    branches:
+      - master # with #262 - ${{{ github.default_branch }}}
+  pull_request:
+    branches:
+      - master # with #262 - ${{{ github.default_branch }}}
+
+jobs:
+
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: lts/*
+    - uses: ipfs/aegir/actions/cache-node-modules@master
+    - run: npm run --if-present lint
+    - run: npm run --if-present dep-check
+
+  test-node:
+    needs: check
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        node: [16]
```

Since we use lts everywhere else:
```suggestion
        node: ['lts/*']
```